Source/Public/Invoke-DockerTests.ps1

```diff
@@ -32,16 +32,25 @@ function Invoke-DockerTests {
     if (!(Test-Path $absoluteTestReportDir -PathType Container)) {
         New-Item $absoluteTestReportDir -ItemType Directory -Force | Out-Null
     }
+    $osType = Find-DockerOSType
+    $dockerSocket = Find-DockerSocket -OsType $osType
+    if ($osType -eq 'windows') {
+        $configs = 'C:/configs'
```

Comment:
Hardcoded `C:/`?

Reply:
This is inside the docker container 😂

Reply:
We maintain this structure test container

Reply:
Ooo. :ok_hand: